packages/renderer/src/lib/ui/ProviderResultPage.svelte

```diff
@@ -200,11 +200,11 @@ function onSeverityClicked(severity: 'critical' | 'high' | 'medium' | 'low' | 's
               class:text-[var(--pd-state-success)]={result.check.status === 'success'}
               ><Fa size="1.1x" class="mt-1" icon={getIcon(result.check)} />
             </span>
-            <div class="font-bold">{result.check.name}</div>
-            <div class="text-[var(--pd-content-text)] text-sm grow text-right">Reported by {result.provider.label}</div>
+            <div class="font-bold" aria-label="{result.check.name}">{result.check.name}</div>
+            <div class="text-[var(--pd-content-text)] text-sm grow text-right" aria-label="Reported by {result.provider.label}">Reported by {result.provider.label}</div>
           </div>
           {#if result.check.markdownDescription}
-            <div class="mt-4">{result.check.markdownDescription}</div>
+            <div class="mt-4" aria-label="{result.check.name} Description">{result.check.markdownDescription}</div>
```

Comment:
this could contain a label denoting a description of the analysis. General, not dynamic.